### ISSUE TITLE
Персонаж стоит на воздухе

### ISSUE THREAD

**Issue by Mazdaywik** [bmstu-iu9/utp2019-7-2d-game#23]
Персонаж стоит не на кубике, а на воздухе. Это так и должно быть или глюк? Это второй уровень, но я наблюдал такое же поведение на первом уровне, но поленился заскриншотить.

![Персонаж стоит на воздухе](https://user-images.githubusercontent.com/1309859/63757688-37b07580-c8c3-11e9-974b-0176a1a41d40.png)

**Comment by Vigorge** [comment]
Да, спасибо, изначально это было из-за системы отслеживания соприкосновений персонажа с окружающими блоками, но сейчас она переписана, и проблема отпала
